### ISSUE TITLE
Expandable/ExpandableGroup focus is not on whole element when expanded

### ISSUE THREAD

**Issue by anselmbradford** [cfpb/capital-framework#760]
Moved from https://github.com/cfpb/cfgov-refresh/issues/1161

## Current behavior

The entire Expandable isn't highlighted when the element has the focus. 
Depicted in the following image : 

![image](https://cloud.githubusercontent.com/assets/1860176/11229219/39fedc92-8d61-11e5-86a3-fc55803e3b9a.png)

## Expected behavior

- focus is around whole expandable when expanded.

## Steps to replicate behavior

1. Click an expandable and note focus area.

**Comment by Scotchester** [comment]
Discussed with Ans and concluded that this isn't actually an accessibility issue, but certainly not ideal from a design perspective.

**Comment by nataliafitzgerald** [comment]
@anselmbradford @Scotchester 
Related issue from Design Manual repo: https://github.com/cfpb/design-manual/issues/438

**Comment by jimmynotjim** [comment]
The focus is still on the target, as it should be from an a11y standpoint, but the focus styles are now fixed. I think we should close this unless something changes in the Design Manual.

<img width="895" alt="screen shot 2018-07-26 at 2 50 34 pm" src="https://user-images.githubusercontent.com/1280430/43285012-4624cc24-90e3-11e8-8d2e-0c262da883ba.png">
